Bugzilla/Search/Saved.pm

```diff
@@ -136,15 +154,20 @@ sub _check_query {
 #########################
 
 sub create {
-  my $class = shift;
-  Bugzilla->login(LOGIN_REQUIRED);
+  my ($class, $params) = @_;
+  my $user  = Bugzilla->login(LOGIN_REQUIRED);
```

Comment:
You do not need to do this in the object modules. The calling code should be the one to handle auth.

my $user = Bugzilla->user;